jabgui/src/main/java/org/jabref/gui/importer/BookCoverFetcher.java

```diff
@@ -55,9 +55,9 @@ public Optional<Path> getDownloadedCoverForEntry(BibEntry entry, Path directory)
     public void downloadCoversForEntry(BibEntry entry, Path directory) {
         Optional<ISBN> isbn = entry.getISBN();
         if (isbn.isPresent()) {
-            final String name = "isbn-" + isbn.asString();
+            final String name = "isbn-" + isbn.get().asString();
             if (findExistingImage(name, directory).isEmpty()) {
-                final String url = getSourceForIsbn(isbn);
+                final String url = getSourceForIsbn(isbn.get());
                 Optional<LinkedFile> file = downloadCoverImage(url, name, directory);
             }
```

Comment:
Please get used to `.map` for optionals. At the end, you can do `forEach` as you don't need the resulting downloaded file at this place.